portlets/src/main/frontend/webpack.dev.js

```diff
@@ -18,6 +18,8 @@ module.exports = merge(common, {
         GamificationInformations: "./GamificationInformations.js",
         earnpoints: "./earnpoints.js",
         spaceleaderboard: "./spaceleaderboard.js",
+        profileStats: "./profileStats/main.js"
```

Comment:
IMO, it's better to define the list of JS files into webpack.common.js

Reply:
Done!